### ISSUE TITLE
error: the following arguments are required: config

### ISSUE THREAD

**Issue by Snowstorm0** [open-mmlab/mmdetection#488]
usage: train.py [-h] [--work_dir WORK_DIR] [--resume_from RESUME_FROM]
                [--validate] [--gpus GPUS] [--seed SEED]
                [--launcher {none,pytorch,slurm,mpi}]
                [--local_rank LOCAL_RANK]
                config
train.py: error: the following arguments are required: config
-------------
how to solve this problem？

**Comment by tunai** [comment]
You need to provide a config file that leads the training script. It details the training hyperparameters, training/val/testing files details, etc. 

Choose one of the config files under mmdetection/configs/ and edit it if you want to train your own dataset.

**Comment by Snowstorm0** [comment]
> 您需要提供一个引导培训脚本的配置文件。它详细介绍了训练超参数、训练/Val/测试文件的细节等。
> 
> 如果您想要训练您自己的数据集，请选择mm检测/信任/配置/下的配置文件之一，并对其进行编辑。
> 
> 最后，你想要这样的东西：
> 
> `./dist_train.sh /your_config_file_dir 1 --validate `(一个GPU)

Thank you for helping me solve this problem.

**Comment by Snowstorm0** [comment]
> You need to provide a config file that leads the training script. It details the training hyperparameters, training/val/testing files details, etc.
> 
> Choose one of the config files under mmdetection/configs/ and edit it if you want to train your own dataset.
> 
> In the end, you want something like this:
> 
> `./dist_train.sh /your_config_file_dir 1 --validate ` (for a single gpu)

Thank you for helping me solve this problem.
